### ISSUE TITLE
Add LazyGroupNorm

### ISSUE THREAD

**Issue by vsey** [pytorch/pytorch#161869]
### 🚀 The feature, motivation and pitch

Hi, I designed a U-Net with LazyConv, LazyLinear, and LazyBatchNorm. When I tried to replace BatchNorm with GroupNorm, I noticed that there is no lazy implementation of GroupNorm. However, this would be useful for my model, as it only supports lazy norms. Therefore, I would like to contribute an implementation for LazyGroupNorm.

**Comment by vadimkantorov** [comment]
Curious, are Lazy modules used in the wild these days?

In transformers library, I think, all dims are just passed down without much issues...
